pkg/ceph/mon/config.go

```diff
@@ -282,7 +281,6 @@ func CreateDefaultCephConfig(context *clusterd.Context, cluster *ClusterInfo, ru
 			OsdPoolDefaultPgpNum:   100,
 			OsdObjectStore:         store,
 			RbdDefaultFeatures:     3,
-			CrushtoolPath:          "",
 			FatalSignalHandlers:    "false",
```

Comment:
i actually had to remove it!  when you call "set crushmap", the incoming crushmap is verified server side by the monitor by shelling out to crushtool.  if its config has the empty path (like we used to have), that will fail with a "file not found": https://github.com/rook/ceph/blob/rook-luminous/src/mon/OSDMonitor.cc#L6339